_posts/0000-01-02-KalennaCodes.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"I am a software developer, financially stable and working with great employees"